frontend/src/Store/Actions/Settings/indexers.js

```diff
@@ -101,8 +101,8 @@ export default {
     sortKey: 'name',
     sortDirection: sortDirections.ASCENDING,
     sortPredicates: {
-      name: function(item) {
-        return item.name.toLowerCase();
+      name: ({ name }) => {
+        return name.toLocaleLowerCase();
```

Comment:
Will this properly handle numbers? If not something we should fix in the future.